use-timescale/data-tiering/tier-data.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tier data to object storage
+excerpt: How to tier data to object storage
+products: [cloud]
+keywords: [data tiering]
+tags: [storage, data management]
+---
+
+import ExperimentalPrivateBeta from 'versionContent/_partials/_early_access.mdx';
+import TieringBeta from 'versionContent/_partials/_cloud-data-tiering-beta.mdx';
+
+# Tier data to object storage
+
+Tier data from primary storage to object storage to save on storage costs. You
+can continue to query a hypertable as normal after migration. All queries,
+including `JOIN`s, work as usual.
+
+This section describes how to tier data in the Timescale console. For
+information about manually tiering and untiering your data, see the
+[self-hosted data tiering][self-hosted-data-tiering] section
+
+## Turn on data tiering
+
+You can turn on data tiering from the Services Overview page in the Timescale console.
+
+<Procedure>
+
+### Turning on data tiering
+
+1.  In the Timescale console, from the `Services` list, click the name of
+    the service you want to modify.
+1.  In the `Overview` tab, locate the `Data tiering` card, and click
+    `Enable data tiering`. Confirm the action.
```

Comment:
this just enables data tiering. Users have to set up a policy or manually tier data for it to actually get moved

This section needs some more material with some examples on how to about this..

Reply:
Ah ha! I didn't know this, thanks! I'll add some stuff here, but will definitely need more content. Is there anything you can point me to to get started?

Reply:
The "[Manually tier a chunk]" and "[[Automate chunk tiering with a data tiering policy](https://docs-dev.timescale.com/docs-datatiering-ga-lana/use-timescale/datatiering-ga-lana/data-tiering/tier-data/#automate-chunk-tiering-with-a-data-tiering-policy)]" sections on this page would be the next step for users.

Should we move those to their own page? Then this section would have a Call to Action link to "tier your first chunk follow the instructions on this page."

Reply:
They were on their own page, then I moved them in to this one with this most recent change. The reasoning being that enabling it in the console is only the first step, you haven't actually got data tiering until you do the manual bit.